ui/components/multichain/account-menu/account-menu.tsx

```diff
@@ -42,11 +42,9 @@ import {
   ///: BEGIN:ONLY_INCLUDE_IF(bitcoin)
   getIsBitcoinSupportEnabled,
   ///: END:ONLY_INCLUDE_IF
-  ///: BEGIN:ONLY_INCLUDE_IF(solana)
   getIsSolanaSupportEnabled,
-  ///: END:ONLY_INCLUDE_IF
   getHdKeyringOfSelectedAccountOrPrimaryKeyring,
-  ///: BEGIN:ONLY_INCLUDE_IF(solana,bitcoin)
+  ///: BEGIN:ONLY_INCLUDE_IF(multichain)
```

Comment:
Nit: Is it safe to move from `solana,bitcoin` to `multichain`? Maybe it's safer to keep `bitcoin`.

Reply:
if it was included before only with `solana` it's safer now to include it anyways right?

Reply:
Yes, makes sense. My comment was wrong because if we change it to `bitcoin`, Solana won't work. We might even want to remove this fence in the future, but it's not a problem to keep it here since it's enabled on all build types.